packages/runner-ct/package.json

```diff
@@ -8,8 +8,10 @@
     "build": "webpack",
     "build-prod": "cross-env NODE_ENV=production yarn build && tsc",
     "clean-deps": "rm -rf node_modules",
+    "cypress:open": "ts-node ../../scripts/start.js --component-testing --project ${PWD}",
+    "cypress:run": "ts-node ../../scripts/start.js --component-testing --run-project ${PWD}",
     "postinstall": "echo '@packages/runner needs: yarn build'",
-    "test-unit": "ts-mocha --config ./test/.mocharc.js --require './test/setup.js'",
+    "test": "ts-node ../../scripts/cypress.js run-ct --project ${PWD}",
```

Comment:
I think that you can wait for #14894 to be merged. I made everything works and run on CI

Reply:
Sure, ok. That PR seems kind of broken (see the PR - many changed files/commits). Can you fix that one up?

Reply:
Getting CI green was a nightmare and I dont want to do it again so I am just gonna merge this as is, but defintiely keen to see what you come up with.  I grabbed the ts-node thing from here: https://github.com/cypress-io/cypress/pull/14930